lib/tasks/import.rake

```diff
@@ -37,5 +37,19 @@ namespace :figgy do
       ingester = JsonIngester.new(json_path: file_path, logger: @logger)
       ingester.ingest!
     end
+
+    desc "Ingest a DSpace asset."
```

Comment:
Are you planning to run like a bunch of these rake task, or is there a loop or something?